diff -urN 26204.orig/dec_if.c 26204/dec_if.c
--- 26204.orig/dec_if.c	2004-12-09 13:45:08.000000000 +0100
+++ 26204/dec_if.c	2005-05-03 18:58:16.654155300 +0200
@@ -34,7 +34,7 @@
    void *decoder_state;       /* Points decoder state       */
 } WB_dec_if_state;
 
-Word16 nb_of_param_first[NUM_OF_SPMODES]=
+const Word16 nb_of_param_first[NUM_OF_SPMODES]=
 {
 	9,  14, 15,
 	15, 15, 19,
diff -urN 26204.orig/if_rom.c 26204/if_rom.c
--- 26204.orig/if_rom.c	2003-08-27 00:00:02.000000000 +0200
+++ 26204/if_rom.c	2005-05-03 18:58:08.716655300 +0200
@@ -1006,7 +1006,7 @@
  * decoder homing frames for all modes
  */
 
-const Word16 *dhf[10] = {
+const Word16 *const dhf[10] = {
    dfh_M7k,
    dfh_M9k,
    dfh_M12k,
diff -urN 26204.orig/typedef.h 26204/typedef.h
--- 26204.orig/typedef.h	2003-08-27 00:00:02.000000000 +0200
+++ 26204/typedef.h	2005-05-03 18:57:06.326030300 +0200
@@ -15,4 +15,4 @@
 typedef double Float64;
 typedef float Float32;
 
-#endif
\ No newline at end of file
+#endif
